### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by EverardoTV**
# Description
adding PodsLibraries for ios development

### CODE REVIEW

**Comment by EverardoTV** [comment]
idk

File: reservaciones/Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
```
Comment (tmario1508) [inline reservaciones/Gemfile]: Remove comment

File: reservaciones/android/app/_BUCK [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# To learn about Buck see [Docs](https://buckbuild.com/).
+# To run your application with Buck:
+# - install Buck
+# - `npm start` - to start the packager
+# - `cd android`
+# - `keytool -genkey -v -keystore keystores/debug.keystore -storepass android -alias androiddebugkey -keypass android -dname "CN=Android Debug,O=Android,C=US"`
+# - `./gradlew :app:copyDownloadableDepsToLibs` - make all Gradle compile dependencies available to Buck
+# - `buck install -r android/app` - compile, install and run application
+#
+
```
Comment (tmario1508) [inline reservaciones/android/app/_BUCK:1]: Code unnecessary, you delete

File: reservaciones/android/app/src/debug/java/com/reservaciones/ReactNativeFlipper.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
+ * directory of this source tree.
+ */
```
Comment (tmario1508) [inline reservaciones/android/app/src/debug/java/com/reservaciones/ReactNativeFlipper.java]: Delete comments

File: reservaciones/android/app/src/main/java/com/reservaciones/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package com.reservaciones;
+
+import com.facebook.react.ReactActivity;
+
+public class MainActivity extends ReactActivity {
+
+  /**
+   * Returns the name of the main component registered from JavaScript. This is used to schedule
+   * rendering of the component.
+   */
```
Comment (tmario1508) [inline reservaciones/android/app/src/main/java/com/reservaciones/MainActivity.java]: More comments, you delete

**Review by tmario1508 — DISMISSED** [review]
Some files have unnecessary commented code

**Review by EverardoTV — COMMENTED** [review]
removing comments

**Review by tmario1508 — APPROVED** [review]

**Pull request closed without merging**